dtbase/backend/api/sensor/routes.py

```diff
@@ -33,8 +33,13 @@ def insert_sensor_type():
     payload = json.loads(request.get_json())
     for k in ["name", "description", "measures"]:
         if not k in payload.keys():
-            raise RuntimeError(
-                f"Must include '{k}' in POST request to /insert_sensor_type"
+            return (
```

Comment:
This is a good idea. For a lot of things we raise an error rather than returning an error through the API, which would be better.